bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -137,6 +140,45 @@ class PresentationToolbar extends PureComponent {
     return addWhiteboardGlobalAccess(whiteboardId);
   }
 
+  handleRenderToolbarPluginItems() {
+    let pluginProvidedButtons = [];
+    if (this.props) {
+      const {
+        whiteboardPluginProvidedItems: ppb,
+      } = this.props;
+      pluginProvidedButtons = ppb;
+    }
+
+    return pluginProvidedButtons?.map((ppb) => {
+      let returnComponent;
+      const ppbId = ppb.id;
+
+      switch (ppb.type) {
+        case PluginSdk.WhiteboardToolbarItemType.BUTTON:
+          returnComponent = (
+            <Button
+              key={ppbId}
+              style={{ marginLeft: '2px' }}
+              label={ppb.label}
+              onClick={ppb.onClick}
+              tooltipLabel={ppb.tooltip}
+            />
+          );
+          break;
+        case PluginSdk.WhiteboardToolbarItemType.SPINNER:
+          returnComponent = (
+            <Spinner
+              key={ppbId}
+            />
+          );
+          break;
+        default:
+          returnComponent = null;
+      }
+      return returnComponent;
+    });
```

Comment:
This method is mentioning "provided buttons", but now its about "provided items", please review it.